dashboard/app/controllers/videos_controller.rb

```diff
@@ -117,4 +115,11 @@ def i18n_params
   prepend_before_action do
     params[:video] &&= video_params
   end
+
+  def merge_and_write
+    if @video.locale == 'en-US'
```

Comment:
Does the view also need to be updated to pass a locale, so current video saves continue to work in the interim?

(might be good to add a test to cover this case, if there isn't one)

Reply:
:+1: for tests!

Reply:
Added a test for what is passed into merge_and_write_attributes_stub. As 'en-US' is the default locale in Video, this "works" without a view change.

Some changes in the levelbuilder create video view will be sent out in a separate PR.

Reply:
Bethany confirmed the schema.rb default gets loaded by Video.new, TIL!


![](https://files.slack.com/files-pri/T039SAH7W-FGE1UHRC0/screenshot_2019-02-22_at_4.15.59_pm.png)